numba/parfors/array_analysis.py

```diff
@@ -2272,13 +2277,13 @@ def _analyze_op_call_builtins_len(self, scope, equiv_set, loc, args, kws):
         return ArrayAnalysis.AnalyzeResult(shape=shape[0], rhs=shape[0])
 
     def _analyze_op_call_numba_parfors_array_analysis_assert_equiv(
-        self, scope, equiv_set, loc, args, kws
+        self, scope, equiv_set, loc, args, kws, varargs
```

Comment:
At some point perhaps these `_analyze_op*` methods could be refactored to just take a dictionary? Probably 80% of the code change in this patch is adding the same extra arg to many methods?

Reply:
Also, methods like `_analyze_numpy_array_like` do not have this extra arg as they are presumably never used or called in that way, however this makes it quite hard to work out whether that's a mistake or by design.